examples/src/java/com/twitter/heron/examples/streamlet/FilesystemSinkTopology.java

```diff
@@ -0,0 +1,141 @@
+//  Copyright 2017 Twitter. All rights reserved.
```

Comment:
I believe a better example is a Copy example where in you read from one File Source and copy it to another File Sink

Reply:
We could do that, but there is a separate example topology that covers sources. Up to you.

Reply:
It's ok for now. When we will have say a twitter connector handy, we could think of converting this example to reading twitter and writing to a file

Reply:
Agreed. That would be very cool. We can iterate on these a lot in the coming weeks.